site/playwright.config.ts

```diff
@@ -1,19 +1,30 @@
 import type { PlaywrightTestConfig } from "@playwright/test";
+import { devices } from "@playwright/test";
 
 const ci = process.env.CI === "true";
 
+const devicesToUse = [
+  "Desktop Chrome",
+  "Desktop Firefox",
+  "Desktop Safari",
+  "iPhone 11 Pro",
+  "Pixel 5",
+];
+
 const config: PlaywrightTestConfig = {
   forbidOnly: ci,
   projects: [
-    {
+    ...devicesToUse.map((deviceName) => ({
       name: "integration",
```

Comment:
Looks like we end up with multiple projects having the same name 🤔 Does Playwright support this?

Reply:
It appears they do, or at least that's the recommended approach [here](https://github.com/microsoft/playwright/issues/13046#issuecomment-1078259500)